parm/config/gfs/config.resources

```diff
@@ -393,13 +393,14 @@ elif [[ "${step}" = "ocnanalrun" ]]; then
    case ${CASE} in
       C384)
         npes=480
-        export memory_ocnanalchkpt="2.8TB"
+        memory_ocnanalrun="128GB"
```

Comment:
This is more memory than the nodes on Hera have, is this okay?

Reply:
It runs ... so yes @CoryMartin-NOAA